ETC.tsv

```diff
@@ -4608,9 +4608,9 @@ ETC_20150317_004785	Ask about the Divine Day
 ETC_20150317_004786	Ask what happened next
 ETC_20150317_004787	I found the lever handle!
 ETC_20150317_004788	I found the information about the flow of horsepower.
-ETC_20150317_004789	Special information can not be found.
+ETC_20150317_004789	Special information could not be found.
```

Comment:
Could is implying past tense, while can is implying present tense. This would be implying that the information could not be found now or in the past. While the original translation implies that it cannot be found currently. If anything it should be changed to "Special information cannot be found."